src/main/java/com/exasol/adapter/dialects/hive/HiveSqlDialect.java

```diff
@@ -104,6 +105,16 @@ public NullSorting getDefaultNullSorting() {
         return NullSorting.NULLS_SORTED_LOW;
     }
 
+    @Override
+    // https://docs.cloudera.com/documentation/enterprise/5-9-x/topics/impala_literals.html#string_literals
```

Comment:
```suggestion
    // https://cwiki.apache.org/confluence/display/Hive/LanguageManual+Types#LanguageManualTypes-StringsstringStrings
    // https://cwiki.apache.org/confluence/display/Hive/CAST...FORMAT+with+SQL%3A2016+datetime+formats
```
Second link explicitly mentions `\'`.